changelogs/unreleased/gh-7231-memtx-hash-idx-gt-iter-deprecation.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/memtx
+
+* Deprecated **HASH** index 'GT' iterator type (gh-7231).
```

Comment:
Nit: Why is **HASH** in bold?

Reply:
To emphasize it's rather a name than an acronym.

Reply:
But all our [documentation](https://www.tarantool.io/en/doc/latest/concepts/data_model/indexes/) refers to index types as CAPS.